internal/host/validator.go

```diff
@@ -1172,8 +1172,14 @@ func (v *validator) printDefaultRoute(c *validationContext, status ValidationSta
 	}
 }
 
-func isNonePlatformMultiNodeInstallation(c *validationContext) bool {
-	return swag.BoolValue(c.cluster.UserManagedNetworking) && !common.IsSingleNodeCluster(c.cluster)
+func shouldValidateDnsResolution(c *validationContext) bool {
+	// Skip DNS resolution checks in IPI network mode
+	if !swag.BoolValue(c.cluster.UserManagedNetworking) {
+		return false
+	}
+	// Only SNO clusters with DNSMasq manifests enabled need to be checked
```

Comment:
The comment should say the opposite (sno with dnsmasq enabled *don't* need to be checked)

Reply:
:face_with_spiral_eyes: fixed